tests/cases/multiple_ubo_update_with_graphics_pipeline.vkscript

```diff
@@ -162,8 +150,8 @@ probe ssbo float 2:3 0  ~= 0.1 0.2  0.3  0.4 \
                            0.5 0.6  0.7  0.8 \
                            0.9 0.10 0.11
 
+uniform ubo 0:0 vec4  0  0.5 0.5 0.35 0.6 \
```

Comment:
This is the one change. The previous code updated in the middle of an element to half way through the second element. I changed Amber to dis-allow this type of operation, it now always requires a full item to be updated. This makes things a lot simpler internally (and this is a really confusing thing to do in a test)